docs/contributing/summer-with-zulip.md

```diff
@@ -54,7 +54,7 @@ materials](https://developers.google.com/open-source/gsoc/resources/manual).
     need help learning, and time-saving tricks.
 
   - If you need feedback from the community / decisions made, ask in the
-    appropriate public stream on [chat.zulip.org](https://chat.zulip.org). Often
+    appropriate public stream on [the Zulip developer community](https://zulip.com/developer-community/). Often
```

Comment:
I think wherever we say "on the Zulip developer community", we should probably make it "in the Zulip developer community".